flutter_cache_manager/pubspec.yaml

```
@@ -1,27 +1,27 @@
 name: flutter_cache_manager
 description: Generic cache manager for flutter. Saves web files on the storages of the device and saves the cache info using sqflite.
-version: 3.3.0
+version: 3.4.0
```

This is not really a new feature, so I would upgrade patch version
```suggestion
version: 3.3.1
```